src/components/layout.tsx

```diff
@@ -13,8 +13,8 @@ interface LayoutProps {
 const Layout: React.FC<LayoutProps> = (props) => {
   return (
     <>
-      <Head {...props} />
       <Header />
+      <Head {...props} />
```

Comment:
Hmm I don't know why we need this change. Was it an accidental commit add?

Reply:
Yes, can undo this but it doesn't affect anything